tests/test_reading_api.py

```diff
@@ -19,8 +19,11 @@ def test_unpack_list():
 @patch("openapi_server.apis.reading_api.client_for_basic_auth")
 def test_get_success(mocked_client):
     mocked_client.return_value.get_judgment_xml.return_value = b"<judgment></judgment>"
+    mocked_client.return_value.user_can_view_unpublished_judgments.return_value = True
```

Comment:
could be worth adding a test to show that when `user_can_view_unpublished_judgments` returns false, the judgement is not shown